### PR TITLE
Restore ‘no app running’ inspector message (#2762)

### PR DESCRIPTION
**Pull request by pq**
Fixes: #2762.

Confirmed this works in 2018.3 and AS 3.2.1.

Happy to add a check for earlier versions if you all think it's prudent?

/cc @jacob314 @devoncarew

### CODE REVIEW

File: src/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -473,6 +481,15 @@ private void updateForEmptyContent(ToolWindow toolWindow) {
     }
 
     toolWindow.setIcon(FlutterIcons.Flutter_13);
+
+    // Display a 'No running applications' message.
+    final ContentManager contentManager = toolWindow.getContentManager();
+    final JPanel panel = new JPanel(new BorderLayout());
+    final JLabel label = new JLabel("No running applications", SwingConstants.CENTER);
```
Comment (jacob314) [inline src/io/flutter/view/FlutterView.java]: nit: JLabel -> JBLabel

**Comment by jacob314** [comment]
lgtm!

**Comment by devoncarew** [comment]
> Happy to add a check for earlier versions if you all think it's prudent?

IntelliJ 2018.1 / AS 3.2 may still not have the fix; we should just keep an eye on this after it lands (the issue was manifested as the Inspector view not switching over to the live content tab when at app ran).

File: src/io/flutter/view/FlutterView.java (hunk 1 above)
Comment (pq) [reply src/io/flutter/view/FlutterView.java]: 👍

**Pull request merged**